azavea/raster-vision
# Fine tune checkpoint output

## Overview

Creates a fine tune checkpoint `.tar.gz` file for TF Object Detection and TF DeepLab backends. These files can be used to further train models using transfer learning or fine tuning.

## Testing Instructions

This was a bit tough to test. The way I did is by modifying the integration tests to output to a non-temporary directory, running, and then using the fine tune checkpoint output file as the pretrained model to run another integration test, and ensure nothing blows up.

Closes #286
